sdk/security/keyvault/azadmin/test-resources-post.ps1

```diff
@@ -90,6 +90,13 @@ if (!$DeploymentOutputs['AZURE_MANAGEDHSM_URL']) {
     exit
 }
 
+Log "Refreshing OIDC token"
+az cloud set -n $Environment
+az login --federated-token $env:ARM_OIDC_TOKEN --service-principal -t $TenantId -u $TestApplicationId
+if ($LASTEXITCODE) { exit $LASTEXITCODE }
+az account set --subscription $SubscriptionId
+if ($LASTEXITCODE) { exit $LASTEXITCODE }
+
```

Comment:
Should this also be added to azkeys? Know we also deploy managed hsm's on a weekly basis there

Reply:
Oh! Yes I'll add it there.